tests/utilities_unit_test.py

```diff
@@ -114,18 +116,55 @@ def test_rotate_coordinates_rel_west():
         wind_directions,
         coordinates
     )
-    np.testing.assert_almost_equal( Y_COORDS, x_rotated[0,0] - np.min(x_rotated[0,0]))
-    np.testing.assert_almost_equal( X_COORDS, y_rotated[0,0] - np.min(y_rotated[0,0]))
+    np.testing.assert_almost_equal(Y_COORDS, x_rotated[0] - np.min(x_rotated[0]))
+    np.testing.assert_almost_equal(X_COORDS, y_rotated[0] - np.min(y_rotated[0]))
     np.testing.assert_almost_equal(
         Z_COORDS + np.min(Z_COORDS),
-        z_rotated[0,0] + np.min(z_rotated[0,0])
+        z_rotated[0] + np.min(z_rotated[0])
     )
 
     wind_directions = np.array([90.0])
     x_rotated, y_rotated, z_rotated, _, _ = rotate_coordinates_rel_west(
+        wind_directions, coordinates
+    )
+    np.testing.assert_almost_equal(X_COORDS[-1:-4:-1], x_rotated[0])
+    np.testing.assert_almost_equal(Y_COORDS, y_rotated[0])
+    np.testing.assert_almost_equal(Z_COORDS, z_rotated[0])
+
+
+def test_reverse_rotate_coordinates_rel_west():
+    # Test that appplying the rotation, and then the reverse produces the original coordinates
+
+    # Test the reverse rotation
+    coordinates = np.array([[x, y, z] for x, y, z in zip(X_COORDS, Y_COORDS, Z_COORDS)])
```

Comment:
This form was used when the coordinates were all `Vec3` classes, but now since they're just an array we use `np.array(list(zip(X_COORDS, Y_COORDS, Z_COORDS)))` for a little less code.